router/connection_maker.go

```diff
@@ -15,12 +15,12 @@ const (
 )
 
 type ConnectionMaker struct {
-	ourself           *LocalPeer
-	peers             *Peers
-	normalisePeerAddr func(string) string
-	targets           map[string]*Target
-	cmdLinePeers      map[string]string
-	actionChan        chan<- ConnectionMakerAction
+	ourself      *LocalPeer
+	peers        *Peers
+	port         int
+	targets      map[string]*Target
```

Comment:
Not something for this PR, but I feel that the readability of the entire ConnectionMaker would be greatly improved if all these different `map[string]...` used appropriate type aliases for the key instead of a plain string e.g.

````
type TCPAddrString string
...
target map[TCPAddrString]*Target
````

thoughts?